.github/workflows/frontend.yml

```diff
@@ -0,0 +1,32 @@
+name: Check search frontend
```

Comment:
It looks like you've got some of this stuff in here, but haven't merged in the actual branch. I can push a merge commit if you like.

Reply:
> * The search box at the top appears to not do anything
This probably relates to changes in how the WeedCOCOs are flattened and imported to ES. When I originally put it together, you could type in things and alter the output

> * We probably want most of the view to be fixed, with the search pane scrollable, so that scrolling down in the search results doesn't move the search pane.
Probably a result of me building this using "Flex" components. I can probably pretty easily change the facet/filter bar to be a fixed.

> * We might want to combine species + role into one beast: "daugus carota (crop)", "guittierezzia sarothrae (weed)". Indeed we might want to establish a similar convention in Weed COCO category names.
I kind of like this idea, because it could help with the situation where you have daugus carota as weeds and drops.

Reply:
> I can probably pretty easily change the facet/filter bar to be a fixed.

On the contrary, I think the results need to be fixed, the facets scrollable.

I don't know flex well, but I'd imagine it could handle some of that.

Reply:
Sorry what I meant was having a fixed width for the facet/filter column, with scrolling. But yes the actual UI should be relatively easy to adjust and improve.